### PR TITLE
Try to fix specs

### PR DESCRIPTION
**Pull request by ivanovaleksey**
Ref #172 

Hi @rmm5t,
I tried to fix specs but I didn't have any experience in CarrierWave gem so I have a lot of troubles.
If you don't mind I leave some of my attempts and comments here.

### CODE REVIEW

File: spec/mongoid_spec.rb [hunk 1]
```diff
@@ -117,7 +117,7 @@ def extension_white_list
         @doc.save!
         @doc.reload
 
-        expect(JSON.parse({:data => @doc.image}.to_json)).to eq("data"=>{"image"=>{"url"=>"/uploads/test.jpeg"}})
+        expect(JSON.parse({:data => @doc.image}.to_json)).to eq("data"=>{"url"=>"/uploads/test.jpeg"})
```
Comment (ivanovaleksey) [inline spec/mongoid_spec.rb]: Serialization to JSON [have changed](https://github.com/carrierwaveuploader/carrierwave/blob/master/lib/carrierwave/uploader/serialization.rb#L14). It is no longer wrapped within `mounted_as`.

File: spec/mongoid_spec.rb [hunk 2]
```diff
@@ -409,7 +409,7 @@ def extension_white_list
     end
 
     after do
-      FileUtils.rm_rf(file_path("uploads"))
+      FileUtils.rm_rf(public_path("uploads"))
```
Comment (ivanovaleksey) [inline spec/mongoid_spec.rb]: It doesn't fix anything but it seems to me like it's `public/uploads` we should remove not `fixtures/uploads`.

File: lib/carrierwave/mongoid.rb [hunk 3]
```diff
@@ -97,7 +97,7 @@ def serializable_hash(options=nil)
 
           self.class.uploaders.each do |column, uploader|
             if (!only && !except) || (only && only.include?(column.to_s)) || (except && !except.include?(column.to_s))
-              hash[column.to_s] = _mounter(column.to_sym).uploader.serializable_hash
+              hash[column.to_s] = _mounter(column.to_sym).uploaders[0].serializable_hash
```
Comment (ivanovaleksey) [inline lib/carrierwave/mongoid.rb]: `Mounter` is now defined in separate file and [supports](https://github.com/carrierwaveuploader/carrierwave/blob/master/lib/carrierwave/mounter.rb#L32) multiple uploaders. My `uploaders[0]` seems to break everything for `carrierwave '< 1'` so I am not sure how to implement in correctly.

**Comment by ivanovaleksey** [comment]
Spec `spec/mongoid_spec.rb:99` would be passed if there were no `reject(&:blank?)` [here](https://github.com/carrierwaveuploader/carrierwave/blob/master/lib/carrierwave/mounter.rb#L29).

Spec `spec/mongoid_spec.rb:417` is failed due to object `changes`.  As I understand Carrierwave now [use](https://github.com/carrierwaveuploader/carrierwave/blob/master/lib/carrierwave/mount.rb#L193) previous changes to remove previous upload. And I found some patch [here](https://github.com/carrierwaveuploader/carrierwave-mongoid/blob/master/lib/carrierwave/mongoid.rb#L40) and [here](https://github.com/carrierwaveuploader/carrierwave-mongoid/blob/master/lib/carrierwave/mongoid.rb#L73) which I think don't go well now.
Once again I am not sure what to do with it. 

Thanks.

**Comment by zdraganov** [comment]
Any activity on that? Is there anyone supporting this gem?

**Comment by ivanovaleksey** [comment]
Hi @zdraganov, I haven't done anything since that time.
I have not so much knowledge about the gem to choose right approach.

**Comment by rmm5t** [comment]
Closing this out in favor of #179. Please feel free to review #179, and if there's anything you think is missing, feel free to open a new PR. 

(Sorry for the delayed maintenance on this project)

**Pull request closed without merging**